src/scripts/index.js

```diff
@@ -0,0 +1,20 @@
+import createTag from "./utils/createTag.js";
+
+//Send request DOM API elements
+const divSend = createTag({ className: "div-send-request" });
+const spanSelectAndInput = createTag({
+  tagName: "span",
+  className: "span-select-and-input",
+});
+const selectSend = createTag({ tagName: "select", tagId: "select-send" });
+const inputSend = createTag({ tagName: "input", tagId: "input-send" });
+const btnSend = createTag({
+  tagName: "button",
+  tagText: "SEND",
+  tagId: "btn-send",
+});
```

Comment:
We prefer styling only using HTML classes, because this is the most flexible way to reuse and maintenance code. BEM -> 'send__submit'

Reply:
Understand, done